app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,50 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+
+    category = benchmark_run.result.keys[0]
```

Comment:
It isn't clear because the result is stored in JSON

https://github.com/ruby-bench/ruby-bench-web/blob/master/app/services/chart_builder.rb#L15-L19